### PR TITLE
Flutter SDK Beta Documentation Edit

### PR DESCRIPTION
**Pull request by alai97**
### What and why?

Prepares the Flutter SDK README for single sourcing into the Docs site with a doc review first.

### How?

Formatting borrowed from iOS Monitoring and Android Monitoring doc pages.

### Review checklist

- [ ] This pull request has appropriate unit and / or integration tests 
- [ ] This pull request references a Github or JIRA issue

### CODE REVIEW

File: packages/datadog_flutter_plugin/README.md [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
+| Android | iOS |  Web | MacOS | Linux | Windows |
+| :-----: | :-: | :---: | :-: | :---: | :----: |
+|   ✅    | ✅  |  🚧   | ❌  |  ❌   |   ❌   |
 
-## Current Datadog SDK Versions
+### iOS
+
+Your iOS Podfile must have `use_frameworks!` (which is true by default in Flutter) and target iOS version >= 11.0.
 
-[//]: # (SDK Table)
```
Comment (fuzzybinary) [inline packages/datadog_flutter_plugin/README.md:13]: Is there any way we can keep this version mapping table, or does it just not render properly at all on the Docs site?

**Review by fuzzybinary — APPROVED** [review]

File: packages/datadog_flutter_plugin/README.md [hunk 2]
```diff
@@ -1,39 +1,38 @@
-<p align="center">
-  <img src="dd_logo.png" width="200">
-</p>
```
Comment (fuzzybinary) [inline packages/datadog_flutter_plugin/README.md:3]: Also, I'm sad to remove the logo. It looks really good on the pub.dev site. Maybe something to look into for the future is a way to mark sections of the file to be excluded from the docs site?

**Review by apigirl — APPROVED** [review]

**Review by alai97 — COMMENTED** [review]
Re-adds the hero image + version mapping table!

File: packages/datadog_flutter_plugin/README.md [hunk 3]
```diff
@@ -2,11 +2,15 @@
   <img src="dd_logo.png" width="200">
```
Comment (alai97) [inline packages/datadog_flutter_plugin/README.md:2]:
```suggestion
  {{< img src="dd-logo.png" alt="Clone Dashboard" style="width:100%;">}}
```

File: packages/datadog_flutter_plugin/README.md [hunk 4]
```diff
@@ -1,12 +1,16 @@
 <p align="center">
-  <img src="dd_logo.png" width="200">
+  {{< img src="dd-logo.png" alt="Datadog logo" style="width:30%;">}}
```
Comment (fuzzybinary) [inline packages/datadog_flutter_plugin/README.md:2]: So... it looks like we might not be able to have this in for the README and out on the documentation site. By using this syntax (I'm guessing so the Docs site works), it breaks the README (and the result on pub.dev) so probably best to just remove it.

**Review by fuzzybinary — APPROVED** [review]

File: packages/datadog_flutter_plugin/README.md (hunk 4 above)
Comment (alai97) [reply packages/datadog_flutter_plugin/README.md:2]: Ok, I'm sorry that we can't replicate the image when we single source it into the docs 😫

Comment (alai97) [reply packages/datadog_flutter_plugin/README.md:2]: We can still keep the image in the main repo README (https://github.com/DataDog/dd-sdk-flutter#readme)!

**Review by xgouchet — APPROVED** [review]

**Pull request merged**